hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/hbase/SparkHoodieHBaseIndex.java

```diff
@@ -145,22 +154,22 @@ public HBaseIndexQPSResourceAllocator createQPSResourceAllocator(HoodieWriteConf
   private Connection getHBaseConnection() {
     Configuration hbaseConfig = HBaseConfiguration.create();
     String quorum = config.getHbaseZkQuorum();
-    hbaseConfig.set("hbase.zookeeper.quorum", quorum);
+    hbaseConfig.set(ZOOKEEPER_QUORUM, quorum);
     String zkZnodeParent = config.getHBaseZkZnodeParent();
     if (zkZnodeParent != null) {
-      hbaseConfig.set("zookeeper.znode.parent", zkZnodeParent);
+      hbaseConfig.set(ZOOKEEPER_ZNODE_PARENT, zkZnodeParent);
     }
     String port = String.valueOf(config.getHbaseZkPort());
-    hbaseConfig.set("hbase.zookeeper.property.clientPort", port);
+    hbaseConfig.set(CLIENT_ZOOKEEPER_CLIENT_PORT, port);
```

Comment:
this constant resolves to `hbase.client.zookeeper.property.clientPort`. can you confirm if it's fully compatible across different hbase versions? could you pls also global search (e.g. `hbase.zookeeper.property.clientPort` ) and ensure nothing breaks?

Reply:
Thank you very much for your help reviewing the code, I will double check.

Reply:
@xushiyan Thank you very much for your reminder! I am looking for the `hbase.zookeeper.property.clientPort` parameter in the Hbase project, this parameter should be `ZOOKEEPER_CLIENT_PORT`

![image](https://user-images.githubusercontent.com/55643692/196857465-3d24cc67-fa9d-4905-9b1d-1f025b034e43.png)

Reply:
The configuration of the Hudi project involving HBase has been checked. There is a hard-coded situation in TestSparkHoodieHBaseIndex.java, which has been modified.